docs/tooling/kubefirst-cli.md

```diff
@@ -0,0 +1,148 @@
+# Kubefirst Command Line
+
+Kubefirst provides a CLI to empower the whole previsioning cluster management process. The basic usage of the CLI is to
+fully create the Kubefirst previsioning process, but there are also many other handful options to backup, validation and
+cluster destroy.
+
+## Kubefirst CLI available commands
+
+- [clean](#clean)
+- [init](#init)
+- [cluster](#cluster-create)
+- [destroy](#destroy)
+- [backupSSL](#backupssl)
+- [restoreSSL](#restoressl)
+- [state](#state)
+
+### clean
+
+Kubefirst creates files, folders and cloud buckets during installation at your environment. This command removes and
+re-create Kubefirst base files. To destroy cloud resources you need to specify additional flags (--destroy-buckets)
+
+```
+Usage:
+kubefirst clean [flags]
+
+Flags:
+--destroy-buckets   destroy buckets created by init cmd
+--destroy-confirm   confirm destroy operation (to be used during automation process)
+-h, --help              help for clean
+--rm-logs           remove logs folder
+```
+
+### init
+
+Init command will prepare the installation, and create the initial resources that will support the installation when 
+`kubefirst cluster create` is be called.
+
+```
+Usage:
+kubefirst init [flags]
+
+Flags:
+--admin-email string        the email address for the administrator as well as for lets-encrypt certificate emails
+--aws-assume-role string    instead of using AWS IAM user credentials, AWS AssumeRole feature generate role based credentials, more at https://docs.aws.amazon.com/STS/latest/APIReference/API_AssumeRole.html
```

Comment:
```suggestion
--aws-assume-role string    Instead of using AWS IAM user credentials, AWS AssumeRole feature generates role based credentials, more at https://docs.aws.amazon.com/STS/latest/APIReference/API_AssumeRole.html.
```